
# TODO: This launch file will only launch those nodes needed for on the car.

# NOTE: If I would ever need more than one "pose_publisher" node, we don't have
# to use anonymous names, but could instead REMAP the names. Maybe anonymous
# naming is still an option, but I haven't found anything in the docs about it
# yet for ROS2.

